SpeechResponder/eddi.zh-CN.json

```diff
@@ -1,1176 +1,1170 @@
-      "default": true
+      "default": true,
+      "name": "Asteroid prospected",
+      "description": "Triggered when using a prospecting drone"
     },
     "Backpack": {
-      "name": "Backpack",
-      "description": "Lists the contents of your backpack when you disembark",
       "enabled": true,
       "priority": 3,
       "responder": true,
       "script": null,
-      "default": true
+      "default": true,
+      "name": "Backpack",
+      "description": "Lists the contents of your backpack when you disembark"
     },
     "Backpack changed": {
-      "name": "Backpack changed",
-      "description": "Triggered when there is any change to the contents of the suit backpack",
       "enabled": true,
       "priority": 3,
       "responder": true,
       "script": null,
-      "default": true
+      "default": true,
+      "name": "Backpack changed",
+      "description": "Triggered when there is any change to the contents of the suit backpack"
     },
     "Blueprint make report": {
-      "name": "Blueprint make report",
-      "description": "Report on how many of a particular blueprint we can make",
       "enabled": true,
-      "priority": null,
+      "priority": 3,
```

Comment:
`priority` has no impact for scripts that are not triggered through the responder.